templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,51 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
```

Comment:
You can take a look at the other `templates/zerver/help` files that are filed under the **Administrating a Zulip organization** in `templates/zerver/help/index.md` for reference.